test/JDBC/input/datatypes/TestSpatialFun-vu-prepare.txt

```diff
@@ -0,0 +1,123 @@
+CREATE TABLE YourTableTemp ( ID INT PRIMARY KEY, PointColumn geometry ); 
+
+INSERT INTO YourTableTemp (ID, PointColumn) VALUES (1, geometry::Point(3.0, 4.0, 4326)), (2, geometry::Point(5.0, 6.0, 4326));
+
+CREATE TABLE YourTableTemp1 ( ID INT PRIMARY KEY, STX geometry ); 
```

Comment:
Please make sure to use all created objects in verify file, via `SELECT` or any other statements

Reply:
resolved